.travis.yml

```diff
@@ -14,8 +15,14 @@ install:
 - pip install "$DJANGO_VERSION" -U
 script:
 - coverage run --source='star_ratings' runtests.py
+- ./run_selenium_tests.sh
```

Comment:
Not strictly related to this PR, but we need to standardise the names of these scripts so that they're consistent. We have:

runtests.sh
runwithcoverage.sh 
run_selenium_tests.sh

I'll create a new issue for this.